apps/frontend/src/pages/not-found/styles.module.css

```
@@ -0,0 +1,47 @@
+.container {
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	color: var(--color-text-primary);
+	background-color: var(--color-background-primary);
+}
+
+.title {
+	margin: 0;
+	font-size: 96px;
+	font-weight: 600;
+	line-height: 116.18px;
+}
+
+.subtitle {
+	margin: 0;
+	font-size: 30px;
+	font-weight: 600;
+	line-height: 36px;
```

use relative values for line-height
```suggestion
	line-height: 120%;
```